website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
Wow. I guess not? Do we process CSS into `public/` at all? I'm not seeing as much in development.

```
/Users/whit537/workbench/cos/osf.io/website/static/css
total 164
drwxr-xr-x  29 whit537  staff    986 Jan  8 15:17 ./
drwxr-xr-x  21 whit537  staff    714 Dec 16 17:24 ../
-rw-r--r--   1 whit537  staff    270 Aug  8 12:06 addon_folderpicker.css
-rw-r--r--   1 whit537  staff    101 Aug  8 12:06 addonsettings.css
-rw-r--r--   1 whit537  staff  21966 Aug  8 12:06 animate.css
-rw-r--r--   1 whit537  staff    115 Dec 29 10:43 autocomplete.css
-rw-r--r--   1 whit537  staff   2536 Aug  8 12:06 bootstrap-xl.css
-rw-r--r--   1 whit537  staff    357 Aug  8 12:06 citations.css
-rw-r--r--   1 whit537  staff    502 Aug  8 12:06 citations_widget.css
-rw-r--r--   1 whit537  staff   1316 Jan  8 15:17 commentpane.css
-rw-r--r--   1 whit537  staff  14827 Dec 16 17:24 fangorn.css
-rw-r--r--   1 whit537  staff    270 Aug  8 12:06 folderpicker.css
drwxr-xr-x   3 whit537  staff    102 Aug  8 12:06 font/
-rw-r--r--   1 whit537  staff  12484 Dec 29 10:43 front-page.css
-rw-r--r--   1 whit537  staff   5242 Aug  8 12:06 hgrid.min.css
-rw-r--r--   1 whit537  staff     84 Aug  8 12:06 iconmap.css
-rw-r--r--   1 whit537  staff    322 Dec 16 17:24 legend.css
-rw-r--r--   1 whit537  staff     88 Jan  5 09:57 nodes-privacy.css
-rw-r--r--   1 whit537  staff   1431 Aug  8 12:06 onboarding.css
drwxr-xr-x  16 whit537  staff    544 Jan  8 15:17 pages/
-rw-r--r--   1 whit537  staff   1919 Dec 29 10:43 prereg.css
-rw-r--r--   1 whit537  staff   7954 Aug  8 12:06 projectorganizer.css
-rw-r--r--   1 whit537  staff    528 Dec 29 10:43 registrations.css
-rw-r--r--   1 whit537  staff   1760 Aug  8 12:06 rubeus.css
-rw-r--r--   1 whit537  staff   1218 Aug  8 12:06 search-bar.css
-rw-r--r--   1 whit537  staff   2939 Dec 16 17:24 share-search.css
-rw-r--r--   1 whit537  staff  13097 Jan  8 15:17 style.css
-rw-r--r--   1 whit537  staff   1621 Dec 16 17:24 typeahead.css
-rw-r--r--   1 whit537  staff    506 Aug  8 12:06 user-addon-settings.css
[osf.io]$
/Users/whit537/workbench/cos/osf.io/website/static/public/css
total 0
drwxr-xr-x   4 whit537  staff  136 Aug  8 12:06 ./
drwxr-xr-x   6 whit537  staff  204 Dec 16 15:08 ../
-rw-r--r--   1 whit537  staff    0 Aug  8 12:06 .gitkeep
drwxr-xr-x  29 whit537  staff  986 Aug  8 12:06 images/
[osf.io]$
```